### PR TITLE
Release #10

### PR DESCRIPTION
**Pull request by AnthonyOliveri**
https://github.ibm.com/Watson/developer-experience/issues/5603

Generated with the following commits:

- API Definitions: [69548d73e9f9b65a549772bdc949087959f9a35e](https://github.ibm.com/Watson/developer-cloud--api-definitions/tree/69548d73e9f9b65a549772bdc949087959f9a35e)
- Watson Swagger Codegen: [17986b7921b9440e72bfb68d3febeb0cda12b7bd](https://github.ibm.com/MIL/watson-swagger-codegen/tree/17986b7921b9440e72bfb68d3febeb0cda12b7bd)

Most of the changes are fairly minor, except for the new tokenization dictionary methods in Discovery.

### CODE REVIEW

File: Source/SpeechToTextV1/SpeechToText.swift [hunk 1]
```diff
@@ -325,13 +334,17 @@ public class SpeechToText {
        parameter. To determine whether a language model supports speaker labels, use the **Get models** method and check
        that the attribute `speaker_labels` is set to `true`. See [Speaker
        labels](https://console.bluemix.net/docs/services/speech-to-text/output.html#speaker_labels).
+     - parameter customizationID: **Deprecated.** Use the `language_customization_id` parameter to specify the
+       customization ID (GUID) of a custom language model that is to be used with the recognition request. Do not
+       specify both parameters with a request.
```
Comment (mkistler) [inline Source/SpeechToTextV1/SpeechToText.swift]: Shouldn't this parameter appear after `model`?  It does in the parameter list.  I think we should probably document the parameters in the same order as they appear in the parameter list.

File: Source/SpeechToTextV1/SpeechToText.swift [hunk 2]
```diff
@@ -731,6 +757,9 @@ public class SpeechToText {
...
```
Comment (mkistler) [inline Source/SpeechToTextV1/SpeechToText.swift]: Same comment as above.  The doc should match the method signature.

**Comment by AnthonyOliveri** [comment]
@mkistler The order of the parameters don't change the quick reference you see when you option-click the method, so I figured it didn't matter. I also like to keep the ever-growing list of hand-edits as minimal as possible. 

However, I just generated the API docs and saw that parameter order **does** matter there, so I'll put the `customizationID` back to where it was.

**Comment by AnthonyOliveri** [comment]
Actually, I just realized that several of those parameters are out of order in the docs versus the method signature, so there's not much point in just fixing this one. Especially since the order will be corrected in the 1.0 release, when `customizationID` (and other parameters) will be reordered.

**Pull request merged**